Editor/Window/ConnectToFleetInput.cs

```diff
@@ -106,25 +109,29 @@ private void RegisterCallBacks(VisualElement container)
 
         private void AssignUiElements(VisualElement container)
         {
-            _fleetNameInput = container.Q<TextField>("AnywherePageCreateFleetInput");
-            _fleetNameDropdownContainer = container.Q<DropdownField>("AnywherePageConnectFleetDropdown");
+            _fleetNameInput = container.Q<TextField>("AnywherePageCreateFleetNameInput");
+            _fleetNameDropdownContainer = container.Q<DropdownField>("AnywherePageConnectFleetNameDropdown");
             _fleetId = container.Q("AnywherePageConnectFleetID");
             _fleetIdText = container.Q<Label>("AnywherePageConnectFleetIDDisplay");
             _fleetStatus = container.Q("AnywherePageConnectFleetStatus");
-            _fleetCreateFoldout = container.Q("AnywherePageCreateFleetTitle");
-            _fleetConnectFoldout = container.Q("AnywherePageConnectFleetTitle");
+            _fleetCreateContainer = container.Q("AnywherePageCreateFleet");
+            _fleetConnectContainer = container.Q("AnywherePageConnectFleet");
             _cancelButton = container.Q<Button>("AnywherePageCreateFleetCancelButton");
         }
 
         private async Task UpdateFleetMenu()
         {
             if (_stateManager.GameLiftWrapper != null)
             {
-                var fleetList = await _fleetManager.ListFleetAttributes();
+                var fleetList = await _fleetManager.ListFleetAttributes(ComputeType.ANYWHERE);
```

Comment:
It looks like this method was renamed in a separate PR that's causing merge conflicts